content/en/service_catalog/manage_entries.md

```diff
@@ -33,14 +33,20 @@ Service color is used in trace visualizations. Click the service type icon to ch
 
 {{< img src="tracing/service_catalog/change_service_color.png" alt="Click the service icon to select a different icon color." style="width:80%;" >}}
 
+## Manage service-related workflows
+[Workflow Automation][14] allows you to automate end-to-end processes across your teams. It integrates with Datadog's Service Catalog to enable dynamic and self-service workflows.
+
 ### Updating service type and language
 With [Service Catalog metadata schema 2.2][5], you can specify the type and language for user-defined services or overwrite the auto-detected type and language for instrumented services. Correctly label the service type and language to help other teams further understand what your services do and how to interact with them. 
 
-### Service Definition Schema (v2.2) (Recommended)
+### Enrich auto-detected services with metadata 
+To specify on-call, source code, or documentation for your services, you can add metadata to any existing services via the UI, APIs, or other automation. 2.2 is the recommended version. To try experimental features, you can opt into the beta program for [schema 3.0][21].
```

Comment:
How should the reader opt into the beta program?

Reply:
addressed